package.json

```diff
@@ -26,8 +26,14 @@
     },
     "homepage": "https://github.com/GPII/gpii-chrome-extension#README.md",
     "scripts": {
-        "test": "node tests/node/all-tests.js",
-        "postinstall": "grunt build"
+        "pretest": "node node_modules/.bin/rimraf reports/* coverage/* && grunt build",
```

Comment:
This line doesn't function correctly in Windows
````
E:\Source\gits\gpii-chrome-extension\node_modules\.bin\rimraf:2
basedir=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")
          ^^^^^^^

SyntaxError: missing ) after argument list
````

Reply:
It seems that the problem on Windows is because rimraf and nyc were accessed through ".bin" instead of directly. See https://github.com/isaacs/rimraf/issues/107#issuecomment-217952571